docs/extensions/readme.md

```diff
@@ -858,6 +858,7 @@ Field Name | Type | Description
 itemName | `string` | Optional (default: `value`). Specifies the name of the property that provides the collection of pageable items.
 nextLinkName| `string` | Required. Specifies the name of the property that provides the next link (common: `nextLink`). If the model does not have a next link property then specify `null`. This is useful for services that return an object that has an array referenced by `itemName`. The object is then flattened in a way that the array is *directly* returned, no paging is used. This provides a better client side API to the end user.
 operationName | `string` | Optional (default: `<operationName>Next`). Specifies the name of the operation for retrieving the next page.
+tokenParamName | `string` | Optional (default: `nextLink`). If you are looking to pass the continuation token through an API parameter to subsequent calls, use this field to specify the name of the parameter that will take the continuation token
```

Comment:
Does it make sense to specify a default value here?  If it does, how about `continuationToken` or something similar?

Reply:
`nextLink` is in a sense the current default value, because say if you want to pass a continuation token into a separate next operation: we currently can do this if the input param name is `nextLink`. I can also remove the default if you think it will confuse users, what are your thoughts?

Reply:
Hmm, perhaps the doc for `nextLinkName` should also be updated to state that it's a next link URL or a continuation token.  For the latter, we can have a "common: `token`" or something so we can distinguish between the two cases.

For `tokenParamName` we could say something like "Contains the name of the continuation token parameter for the case where `nextLinkName` contains a continuation token instead of a URL."

Reply:
I like that, thanks @jhendrixMSFT !